### ISSUE TITLE
[BUG] Apple M1 build issues

### ISSUE THREAD

**Issue by hammady** [bitfocus/companion#1936]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When building using the Dockerfile on master, I get 2 categories of errors. They are confirmed to be related to Apple M1:
1. `sharp` in `package.json` requires version 0.29+ to support Apple M1. The current version in master is 0.28. See [this](https://sharp.pixelplumbing.com/install#apple-m1) link. To fix that, one can do `apt-get install libvips`, but it will bump several libraries along which are not tested.
2. `node-sass` does not have a prebuilt binary on M1, so it invokes `node-gyp` which requires Python 3.6. The version included with the official `node:14` base docker image is 3.5.3. See this [PR](https://github.com/bitfocus/companion/pull/1935) for a suggested fix. This is not really a bug in itself, but a Dockerfile fix or a wiki article for the required dpendencies can be added.

### Steps To Reproduce

1. Check out the latest master branch
2. Build using docker on an Apple M1 machine
3. Observe build errors

### Expected Behavior

No build errors.

### Environment (please complete the following information)

```markdown
- OS: MacOS 12.0.1
- Browser: N/A
- Companion Version: 2.2.0 beta (master commit 762a6e94892e053dc56e9f2d339d1f746428a0a7).
```

**Comment by Julusian** [comment]
While yes the version of sharp doesnt 'support' macos M1, but thats in only in that they dont have a prebuilt binary published. It does work if you have the stuff setup to be able to compile it. Which is true of many of our dependencies for macos on M1 yet. Im pretty sure that node 14 has to be compiled too. So while it would be nice to properly support M1 native development, I don't think it is an easy/quick fix right now (we should probably update node to v16) but will be done in 2.3/3.0

The real problem you are seeing in docker is fixed by sharp 0.30, but can also be solved by switching the docker image to use `node:14-bullseye` instead of `node:14`, as the sharp binaries require a newer version of glibc than the docker image uses, causing the failure.

`node-sass` again yeah it doesnt have a prebuild for either macos or linux arm64. That isnt something we can do anything about right now, as there isnt a newer version which does support it. But with the same docker base image change that makes python 3.9 available for node-gyp, so it happily builds

**Comment by hammady** [comment]
Using `node:14-bullseye` seems to have fixed it from aac62e93115b65d140d285345da042b28f0e0d74.
